src/app/tabs/run-tab.js

```diff
@@ -20,14 +20,17 @@ var noInstancesText = yo`<div class="${css.noInstancesText}">0 contract Instance
 
 var pendingTxsText = yo`<span></span>`
 
-function runTab (container, appAPI, appEvents) {
-  var events = new EventManager()
+function runTab (api = {}, events = {}, opts = {}) {
```

Comment:
here too, don't update the name or change every occurence

Reply:
the idea is to change every module signature to `(api = {}, events = {}, opts = {})` according to your PR to update best practices.
Personally I think it matters that the arguments are named the same in every module, so that people get used to rely on `api`, `events` and `opts` and not have different name in each module.

I see this PR as an opportunity to come closer to that goal